autoprotocol/container.py

```diff
@@ -253,10 +253,17 @@ def set_volume(self, vol):
         v = Unit(vol)
         max_vol = self.container.container_type.true_max_vol_ul
         if v > max_vol:
+            containerIdInfo = ""
+            if self.container.id is not None and len(self.container.id) > 0:
```

Comment:
I think we can simplify this to:

```suggestion
            containerIdInfo = ""
            if self.container.id:
```